task.go

```diff
@@ -301,6 +305,52 @@ func (task *Task) Modify(cmdLine CmdLine) {
 	}
 }
 
+func (task *Task) IsResolved() bool {
+	for _, nonResolvedStatus := range NON_RESOLVED_STATUSES {
+		if task.Status == nonResolvedStatus {
+			return false
+		}
+	}
+	return true
+}
+
+func generateUrgency(task Task) int {
```

Comment:
We might prefer `computeUrgency` or `determineUrgency`, to imply that it's deterministic. This also looks like it should be unit tested, since it's a bit complex

Reply:
I changed the name and added some unit tests.  I also added a make task to generate code coverage so I could make sure I had all cases covered.